.pylintrc

```diff
@@ -38,7 +38,7 @@ load-plugins=linter_plugin
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=fixme,locally-disabled,abstract-class-not-used,bad-continuation,too-few-public-methods,no-self-use,invalid-name
+disable=fixme,locally-disabled,abstract-class-not-used,bad-continuation,too-few-public-methods,no-self-use,invalid-name,too-many-instance-attributes
```

Comment:
We've had to disable that pylint warning 14 times already in our code.  We have plenty of intelligent humans to pay attention to the elegance or otherwise of our code; we don't need foolish robots as well :)